<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiniCADClass</class>
 <widget class="QMainWindow" name="MiniCADClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniCAD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MiniCADWidget" name="showWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>841</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="modelBox">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>模式</string>
    </property>
    <widget class="QRadioButton" name="radioButton_model2D">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2D模式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_model3D">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3D模式</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="sweepBox">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>110</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>挤出</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>143</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="sweepLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_sweep_y"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_sweep_z"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="Label_1">
        <property name="text">
         <string> X：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_sweep_l"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>L：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_sweep_x"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string> Z：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string> Y：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_sweep">
        <property name="text">
         <string>Sweep</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="showBox">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>300</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>显示</string>
    </property>
    <widget class="QCheckBox" name="checkBox_lineShow">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>线框显示</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_volumeShow">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>实体显示</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QListView" name="solidList">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>410</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MiniCADWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">MiniCADWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MiniCAD.qrc"/>
 </resources>
 <connections/>
</ui>
